lib/yahoo-finance.rb

```diff
@@ -117,6 +117,9 @@ def quotes(symbols_array, columns_array = [
       ret = []
       symbols_array.each_slice(SYMBOLS_PER_REQUEST) do |symbols|
         read_quotes(symbols.join("+"), columns_array).map do |row|
+          if options[:na_as_nil]
+            row.each { |item| item[-1] = nil if item[-1] == 'N/A' }
```

Comment:
why the `item[-1]`? Do you want this to always handle just the *last* column data? If so, why?